
message RpcMessage {
    optional uint32 head_code = 1 [default = 0];
    optional string body_msg = 2 [default = ""];
}
